pom.xml

```diff
@@ -470,6 +494,7 @@
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-mapreduce</artifactId>
           </exclusion>
+          <!--
```

Comment:
@heuermh I saw you'd excluded these in https://github.com/bigdatagenomics/adam/blame/master/pom.xml#L473-L484. These come in from a provided dependency, so I don't think we should exclude them. Excluding them causes test failures at runtime when writing Parquet via Spark SQL.